build.gradle

```diff
@@ -28,6 +28,10 @@ configurations.all {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter'
 	implementation "org.springframework.shell:spring-shell-starter-jna"
+	implementation 'org.kohsuke:github-api:1.301'
```

Comment:
Put versions in `ext`, actual versioned dep to `dependencyManagement` and then `implementation` can be without version.